multiwallet_test.go

```diff
@@ -0,0 +1,31 @@
+package dcrlibwallet_test
+
+import (
+	"os"
+	"testing"
+	"testing/quick"
+
+	. "github.com/raedahgroup/dcrlibwallet"
+)
+
+// canUseDir returns true if the program can create the directory
+// It will create the directory if it can
+func canUseDir(directory string) bool {
+	return os.MkdirAll(directory, os.ModePerm) == nil
+}
+
+// TestNewMultiWalletPath checks that NewMultiWallet returns an error for invalid
+// paths or no error for valid paths
```

Comment:
```suggestion
// paths
```
Valid paths may still return an error.